questions/playerkills.md

```diff
@@ -6,7 +6,27 @@ _Related: [Detect Player Deaths](/wiki/questions/playerdeaths)_
 
 In Java this is easy, as there is a whole lot of [scoreboard objectives criteria](https://minecraft.wiki/Scoreboard#Criteria) for every entity you can kill in the game, using the format `minecraft.killed:minecraft.<entity>`, where `<entity>` is a valid type of entity.
 
-Alternatively you can use a [custom advancement using the `player_killed_entity` advancement trigger](https://minecraft.wiki/wiki/Advancement/JSON_format#minecraft:player_killed_entity) if you're using a datapack.
+Alternatively you can use a [custom advancement using the `player_killed_entity` advancement trigger](https://minecraft.wiki/wiki/Advancement/JSON_format#minecraft:player_killed_entity) if you're using a datapack that looks like this:
```

Comment:
The link seems to highlight too many words. I think it's enough to highlight the link for [`player_killed_entity`](https://minecraft.wiki/wiki/Advancement/JSON_format#minecraft:player_killed_entity).
I would also add examples of usage. And also clarify that the scoreboard method for command blocks cannot check any NBT data, including the tag of the killed mob. But this can be done using advancement (and give an example).

Reply:
So I change the advancement to this?

```
{
  "criteria": {
    "requirement": {
      "trigger": "minecraft:player_killed_entity",
      "conditions": {
        "entity": {
          "type": "minecraft:pig",
          "nbt": "{Tags:[\"example\"]}"
        }
      }
    }
  },
  "rewards": {
    "function": "example:kill_entity"
  }
}
```

Reply:
It’s OK now?